frontend/pages/hosts/HostDetailsPage/SoftwareVulnerabilities/SoftwareVulnerabilities.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import FleetIcon from "components/icons/FleetIcon";
+import softwareInterface from "interfaces/software";
+
+const baseClass = "software-vulnerabilities";
+
+class SoftwareVulnerabilities extends Component {
+  static propTypes = {
+    softwareList: PropTypes.arrayOf(softwareInterface),
+  };
+
+  render() {
+    const { softwareList } = this.props;
+
+    const vulsList = [];
+
+    const vulnerabilitiesListMaker = () => {
+      softwareList.forEach((software) => {
+        if (software.vulnerabilities) {
+          software.vulnerabilities.forEach((vulnerability) => {
+            vulsList.push({
+              name: software.name,
+              cve: vulnerability.cve,
+              details_link: vulnerability.details_link,
+            });
+          });
+        }
+      });
+    };
+
+    vulnerabilitiesListMaker();
+
+    const renderVulsCount = (list) => {
+      if (list.length === 1) {
+        return "1 vulnerability detected";
+      }
+      if (list.length > 1) {
+        return `${list.length} vulnerabilities detected`;
+      }
```

Comment:
The second if statement is not needed since you're using `return` in the first one.
```suggestion
      return `${list.length} vulnerabilities detected`;
```